server/lib/nl/page_config_next.py

```diff
@@ -82,6 +72,15 @@ def build_page_config(
   all_svs = list(all_svs)
   sv2name = utils.get_sv_name(all_svs)
 
+  # Add a human answer to the query
+  try:
+    desc = lib_desc.build_category_description(uttr, sv2name)
+    if desc:
+      category.description = desc
+  except Exception as err:
+    logging.warning("Error building category description", err)
```

Comment:
nit: should we add `utils.update_counter(uttr.counters, 'failed_category_description_build', str(err))` or something?

Reply:
done